app/package.json

```diff
@@ -108,9 +108,9 @@
     "test:e2e:ui": "playwright test --ui",
     "dev": "pnpm run dev:server & pnpm run build:static && pnpm run build:relay && vite",
     "dev:ui": "pnpm run build:static && pnpm run build:relay && vite",
-    "dev:server": "source .env && tox r -e phoenix_main -- --dev serve",
+    "dev:server": "source .env && phoenix --dev serve",
     "dev:server:test": "node ./tests/utils/testServer.mjs",
-    "dev:server:init": "tox r -e phoenix_main -- --dev serve --with-trace-fixtures=llama_index_rag --with-projects=demo_llama_index --force-fixture-ingestion",
+    "dev:server:init": "phoenix --dev serve --with-trace-fixtures=llama_index_rag --with-projects=demo_llama_index --force-fixture-ingestion",
```

Comment:
It seems like `tox` does not pick up env vars

Reply:
oh interesting, is this going to effect us in other ways, being out of sync with how things are run in ci? I haven't noticed that but not sure if i'm relying on any env vars

Reply:
Happy to change back. It makes it difficult to run with setting, though.